app/controllers/api/v1/book_suggestion_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1
+    class BookSuggestionController < ApplicationController
+      def index
+        render json: BookSuggestion.all
+      end
+
+      def create
+        @book = BookSuggestion.new(book_params)
```

Comment:
you should use `BookSuggestion.create` and use `@book.valid?`
internally create calls new then save anyway:
```ruby
  def create(attributes = nil, options = {}, &block)
    if attributes.is_a?(Array)
      attributes.collect { |attr| create(attr, options, &block) }
    else
      object = new(attributes, options, &block)
      object.save
      object
    end
```
if you use `valid?` method you get the errors and you can use this in the response